llama_stack/distribution/build_conda_env.sh

```diff
@@ -9,6 +9,9 @@
 LLAMA_MODELS_DIR=${LLAMA_MODELS_DIR:-}
 LLAMA_STACK_DIR=${LLAMA_STACK_DIR:-}
 TEST_PYPI_VERSION=${TEST_PYPI_VERSION:-}
+# This is necessary when installing PyTorch via uv since it's likely to time out
+# Reference: https://github.com/astral-sh/uv/pull/1694
+UV_HTTP_TIMEOUT=500
```

Comment:
this is in seconds?

Reply:
it would be good to make this overridable by doing `UV_HTTP_TIMEOUT=${UV_HTTP_TIMEOUT:-500}` (or maybe use a smaller default?

Reply:
This would be a good default based on my trials. Just made it configurable.